src/neptune_api/api/data_ingestion/check_request_status.py

```diff
@@ -13,7 +13,7 @@
     AuthenticatedClient,
     Client,
 )
-from ...models.ingest_response import IngestResponse
+from ...proto.neptune_pb.ingest.v1.request_status_pb2 import RequestStatus
```

Comment:
Is it possible to replace such cases with an explicit import like `package1.package2.etc`?

Reply:
Yes, this one shouldn't be a problem